index.html

```diff
@@ -0,0 +1,116 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/04e92db7ee.js" crossorigin="anonymous"></script>
+    <title>Ryan's profile page</title>
+    <link rel="stylesheet" href="style.css ">
+</head>
+
+<body>
+
+    <header>
+        <h1> My profile page </h1>
+        
+        <div class="welcome-message"> 
+            <em>
+                Welcome to Ryan's profile
+            </em> 
+        </div>
+    </header>
+
+    <ul class="topnav">
+        <li> <a class= "active" href="#Home">Home</a></li>
+        <li><a href="#contact">Contact</a></li>
+        <li><a href="#about">About</a></li>
+        </ul>
+    
+
+    <div class="flex-container">
+        <div class="image-container"> 
+            <img src="images/Cartoon-ryan.jpg" class="ryan-image">
+            <div>About Me</div>
+        </div>
+        <div class="text-container">
+            <article>
+                <p>
+                   <ul class = "no-bullet-list"> 
+                       <li>I'm 31 (32 in june). I love to travel; few of my favourite places to go are Portugal and Ireland (Republic). 
+                    The type of music i listen to the most is alternative rock, some hip hop and just standard pop stuff.</li>
+                    
+                   <li> I also play the guitar the obviious choice of guitar icons i look up to are hendrix, 
+                    John frusciante, Johnny greenwood. </li>
+                    
+                    <li> I wouldnt say im a gamer but i play few games - halo, fortnite, fifa, final fantasy, witcher, all the
+                    good stuff! </li>
+                    
+                    <li>My favourite films - bladerunner (both), arrival, the shining, Lord of the rings.
+                    Massive Lord of the rings nerd, read the books, have the 4k extended remasters, read some of the
+                    wider Tolkien works and follow a lot of the lore channels on YouTube.
+                </li> </ul>
+                </p>
+            </article>
+        </div>
+    </div>
+
+    <div class="flex-container">
+        <div class="text-container" id="b">
+            <article>
+                <p>
+                    <ul class="no-bullet-list">
+
+                    <li>My current goal short term goal is to become a compitent developer,to be succesful in general in the field i choose to work. </li>
+                    
+                    <li>To be happy in life and enjoy all the little things.</li>
+                
+                    <li>So far the plans i have made to become a developer are in motion as i have enrolled in the SoF bootcamp to make this goal a reality.</li>
+
+
+
+                </p>
+            </article>
+        </div>
```

Comment:
No need for the `div` container here - make the `article` element the text container and cut down on the amount of HTML in your site.